### PR TITLE
docs: added new shields

### PR DESCRIPTION
**Pull request by ashish-patwal**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines 
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

- [x] feature update

* **What is the current behavior?** (You can also link to an open issue here)

- #313

* **What is the new behavior (if this is a feature change)?**

- [x] added new badges including maven , pypi packages repo and coverage and other links .

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

- [x] No

* **Other information**:

### CODE REVIEW

**Review by imotai — APPROVED** [review]
LGTM

**Review by dl239 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -5,6 +5,13 @@
 [![docker pulls](https://img.shields.io/docker/pulls/4pdosc/openmldb.svg)](https://hub.docker.com/r/4pdosc/openmldb)
 [![slack](https://img.shields.io/badge/Slack-Join%20Slack-blue)](https://join.slack.com/t/hybridsql-ws/shared_invite/zt-ozu3llie-K~hn9Ss1GZcFW2~K_L5sMg)
 [![discuss](https://img.shields.io/badge/Discuss-Ask%20Questions-blue)](https://github.com/4paradigm/OpenMLDB/discussions)
+![coverage](https://img.shields.io/codecov/c/github/4paradigm/OpenMLDB/main)
```
Comment (aceforeverd) [inline README.md:8]: use this one, we need the badge clickable
 `[codecov](https://codecov.io/gh/4paradigm/OpenMLDB)`

File: README.md [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+[![release](https://img.shields.io/github/v/release/4paradigm/OpenMLDB?color=lime)](https://github.com/4paradigm/OpenMLDB/releases)
+[![license](https://img.shields.io/github/license/4paradigm/OpenMLDB?color=orange)](https://github.com/4paradigm/OpenMLDB/blob/main/LICENSE)
+[![gitee](https://img.shields.io/badge/Gitee-mirror-lightyellow)](https://gitee.com/paradigm4/OpenMLDB)
+[![maven](https://img.shields.io/badge/maven-pkg-green)](https://mvnrepository.com/artifact/com.4paradigm.openmldb/openmldb-batch)
```
Comment (aceforeverd) [inline README.md:12]: maven icon should show the latest version: `Maven Central`
you can keep the link addreess

File: README.md [hunk 3]
```diff
@@ -5,6 +5,13 @@
...
+[![maven](https://img.shields.io/badge/maven-pkg-blue)](https://mvnrepository.com/artifact/com.4paradigm.openmldb/openmldb-jdbc)
+[![pypi](https://img.shields.io/badge/pypi-pkg-blue)](https://pypi.org/project/openmldb/)
```
Comment (aceforeverd) [inline README.md:14]: same as maven: `PyPI`

**Review by aceforeverd — CHANGES_REQUESTED** [review]

**Comment by ashish-patwal** [comment]
- PTAL for requested changes  
- #355

**Pull request merged**